### PR TITLE
Fix #4648: latex: Now "rubric" elements are rendered as unnumbered section title

### LINKED ISSUE #4648: "rubric" should create an unnumbered \paragraph in latex mode
I use `.. rubric` to create a heading with no number and no entry in TOC. It works fine in HTML, but in the latex output I get something like:

```
1.1.3 Normal heading

Lorem ipsum...

1.1.3.0.0.1 "Rubric" heading
```

which makes no sense. I wanted it unnumbered and got a full set of numbers.

I think `.. rubric` should just create `\paragraph*{...}` instead of `\paragraph{...}`. (Changing "secnumdepth" is not an option, as I have other paragraphs (and subparagraphs) that should be numbered.)

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Bugfix

### Purpose
- refs: #4648

### CODE REVIEW

File: sphinx/writers/latex.py [hunk 1]
```diff
@@ -1252,7 +1252,7 @@ def visit_rubric(self, node):
         if len(node.children) == 1 and node.children[0].astext() in \
            ('Footnotes', _('Footnotes')):
             raise nodes.SkipNode
-        self.body.append('\\paragraph{')
+        self.body.append('\\paragraph*{')
```
Comment (tk0miya) [inline sphinx/writers/latex.py]: In this PR, I did not add any migration pass. I suppose nobody want to keep the behavior.

Comment (tk0miya) [inline sphinx/writers/latex.py]: Note: docutils uses `\subsubsection*` to rubric.
http://repo.or.cz/docutils.git/blob/HEAD:/docutils/docutils/writers/latex2e/__init__.py#l634

Comment (jfbu) [reply sphinx/writers/latex.py]: +1 for `\subsubsection*`.

**Review by jfbu — APPROVED** [review]
I think `\subsubsection*` would make better sense than `\ paragraph*`. Of course either one is better than `\paragraph`.

File: sphinx/writers/latex.py (hunk 1 above)
Comment (tk0miya) [reply sphinx/writers/latex.py]: updated

**Review by jfbu — APPROVED** [review]
LGTM

**Comment by tk0miya** [comment]
Thank you for quick reviewing!

**Pull request merged**